gke-deploy/doc/gke-deploy_apply.md

```diff
@@ -34,12 +34,12 @@ gke-deploy apply [flags]
 
 ```
   -c, --cluster string     Name of GKE cluster to deploy to.
-  -f, --filename string    Configuration file or directory of configuration files to use to create Kubernetes objects (file or files in directory must end with ".yml" or ".yaml").
+  -f, --filename string    Configuration file or directory of configuration files to use to create Kubernetes objects (file or files in directory must end with ".yml" or ".yaml"). The file or directory could also be a GCS url.
   -h, --help               help for apply
   -l, --location string    Region/zone of GKE cluster to deploy to.
   -n, --namespace string   Namespace of GKE cluster to deploy to. If omitted, the namespace(s) specified in each Kubernetes configuration file is used.
   -p, --project string     Project of GKE cluster to deploy to. If this field is not provided, the current set GCP project is used.
-  -R, --recursive          Recursively search through the configuration directory for all yaml files.
+  -R, --recursive          Recursively search through the configuration directory for all yaml files. It also applies to a GCS url.
```

Comment:
```suggestion
  -R, --recursive          Recursively search through the configuration directory for all YAML files. It also applies to a GCS url.
```

Reply:
I know this was already there before you came to it, Chris. Could you change all instances of "yaml" in paragraphs to "YAML" (obviously leaving any "`.yaml`" examples lower case).

Reply:
Done.